From 7bd299528df5c8f0a7cbc1fce1d1910a8ded37e8 Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Tue, 9 May 2023 14:32:39 +0530
Subject: [PATCH 208/281] arm64: dts: qcom: ipq9574: Adding build support for
 various RDPs

Adding build support for the following RDPs
rdp 449
rdp 453
rdp 457
rdp 458
rdp 459

Change-Id: Ie9a88fc2e89f936cd5a8af58480ef67407781ab8
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 arch/arm64/boot/dts/qcom/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 4e71a679c969..a9fa4c475065 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -11,9 +11,14 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c2.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp418.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp433.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp449.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp453.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp454.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp455.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp456.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp457.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp458.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp459.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp467.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp469.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-alcatel-idol347.dtb
-- 
2.17.1

